numpy/core/tests/test_multiarray.py

```diff
@@ -346,17 +346,8 @@ def test_int_subclassing(self):
 
         numpy_int = np.int_(0)
 
-        if sys.version_info[0] >= 3:
-            # On Py3k int_ should not inherit from int, because it's not
-            # fixed-width anymore
-            assert_equal(isinstance(numpy_int, int), False)
-        else:
-            # Otherwise, it should inherit from int...
-            assert_equal(isinstance(numpy_int, int), True)
-
-            # ... and fast-path checks on C-API level should also work
-            from numpy.core._multiarray_tests import test_int_subclass
-            assert_equal(test_int_subclass(numpy_int), True)
+        # int_ doesn't inherit from int, because it's not fixed-width
+        assert_equal(isinstance(numpy_int, int), False)
```

Comment:
```suggestion
        assert not isinstance(numpy_int, int)
```
But its good as is, should not do too much general cleanup.

Reply:
I changed it to the form all other tests use.

Reply:
Thanks for all the suggestions.